.github/workflows/arm-cd.yml

```diff
@@ -0,0 +1,28 @@
+name: ARM CD
```

Comment:
Can you add Apache copyright header to all these files please? It seems the internal CI requires them.

Reply:
added copyright header